lib/ohai/plugins/linux/platform.rb

```diff
@@ -32,12 +32,24 @@ def os_release_file_is_cisco?
     return false unless File.exist?('/etc/os-release')
     os_release_info = File.read('/etc/os-release').split.inject({}) do |map, key_value_line|
       key, _separator, value = key_value_line.partition('=')
-      map[key] = value
+      # strip leading/trailing quotes
+      map[key] = value.gsub(/\A"|"\Z/, '')
       map
     end
-    if os_release_info['CISCO_RELEASE_INFO'] && File.exist?(os_release_info['CISCO_RELEASE_INFO'])
-      os_release_info
+    cisco_release_info = os_release_info['CISCO_RELEASE_INFO']
```

Comment:
Agreed that it's not obvious, however...

```
[root@n9k-108 guestshell]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

CISCO_RELEASE_INFO=/etc/shared/os-release
```

`'CISCO_RELEASE_INFO'` is the key whose value points to a file where the Cisco release info can be found. It's probably too late to change this string in the image.

`cisco_release_info` is a hash of keys to values as loaded from this file. I don't think it makes sense to change this name, personally.